server/sonar-server/src/main/java/org/sonar/server/authentication/ws/AuthenticationWs.java

```diff
@@ -32,6 +32,7 @@ public void define(Context context) {
     controller.setDescription("Handle authentication.");
 
     defineLoginAction(controller);
+    defineLogoutAction(controller);
```

Comment:
it's not classic WS, that's why you don't declare them via a marker interface? Couldn't we still use a common `WsAuthenticationAction` marker interface?